go.mod

```diff
@@ -287,9 +288,6 @@ require (
 	github.com/bep/godartsass/v2 v2.1.0 // indirect
 	github.com/bep/golibsass v1.2.0 // indirect
 	github.com/cespare/xxhash/v2 v2.3.0 // indirect
-	// In later at least v0.7.1, lipgloss changes its terminal detection
-	// which breaks most of our CLI golden files tests.
```

Comment:
Comment got removed, if `go mod tidy` will remove comments then this should go into it's own require declaration to avoid being tidied

Reply:
Nah, I removed the comment since it's super outdated